README.md

```diff
@@ -201,6 +201,119 @@ The script `load.py` (requires Python 3) can be used to restore from such JSON d
 
 The optional parameter `--no-commits` can be added to skip those entries that
 look like a commit ID. Note that this might break previous permalinks.
+
+# CONNECTING JAVA DEBUGGER WITH DOCKER COMPOSE USING INTELLIJ IDE
+
+Connecting to a java debugger can be broken down into the following steps:
+### 1. Run the application using docker-compose
+Create a Dockerfile_debug at the root of the directory with the same content as the Dockerfile of the application
+
+Create a docker-compose-debug.yml file with the same content as the docker-compose.yml file at the root of the directory replacing this in the docker-compose-debug.yml
```

Comment:
Instead of `docker-compose-debug.yml`, maybe we would be able to use `docker-compose.override.yml`? I think the docs already mention it for running in development mode, so users & devs can just use that same file and add the extra debug steps if needed.